osbs/cli/main.py

```diff
@@ -559,7 +559,7 @@ def cli():
                               help="build architecture")
     build_parser.add_argument("-u", "--user", action='store', required=True,
                               help="prefix for docker image repository")
-    build_parser.add_argument("-c", "--component", action='store', required=True,
+    build_parser.add_argument("-c", "--component", action='store', required=False,
                               help="name of component")
```

Comment:
This is where you could put help text like "not used; use com.redhat.component Dockerfile label".